### PR TITLE
All card components completed

### PR DESCRIPTION
**Pull request by yogesh-**
I made total of 6 cards
- card with badges
- card with dismiss
- card with text overlay
- Text only card
- Horizontal card
- Card with shadow

https://user-images.githubusercontent.com/7193961/152439481-60fc0e24-9122-4077-a4a3-1012c5aa6208.mp4

### CODE REVIEW

**Review by ashishnnnnn — COMMENTED** [review]
Code Seems fine to me :+1:

File: css/card.css [hunk 1]
```diff
@@ -0,0 +1,113 @@
+.card-container{
+    border: 0.2rem solid blueviolet;
```
Comment (sharmayatendra) [inline css/card.css]: avoid giving hard coded color

File: css/card.css [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+}
+
+.card-tag,.text-overlay{
+    width: 8rem;
+    position: absolute;
+    right: 2rem;
+    top:3rem;
+    background: red;
+    color: white;
+    font-weight: bold;
```
Comment (sharmayatendra) [inline css/card.css]: same as above : use color variables

File: documentation.html [hunk 3]
```diff
@@ -452,29 +452,258 @@ <h2 class="h2-style">Floating Action Button</h2>
...
+                        style="width: 873px; height: 420px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (sharmayatendra) [inline documentation.html:456]: avoid inline styling keep HTML & CSS separate

File: documentation.html [hunk 4]
```diff
@@ -452,29 +452,258 @@ <h2 class="h2-style">Floating Action Button</h2>
...
+                    </div>
 
-            <!-- card -->
+                </div>
+
+                <!-- Cards with text overlay -->
+
+                <div class="card3">
+                    <h2 class="h2-style">Cards with Text Overlay</h2>
+                    <p class="p-style"> Cards with text overlay allows us to place an important text on the image mostly to make a statement.
```
Comment (sharmayatendra) [inline documentation.html:556]: can use good classnames like card-heading,card-description or more generic

File: documentation.html [hunk 5]
```diff
@@ -483,6 +712,19 @@ <h2 class="h2-style">Floating Action Button</h2>
...
+
+
+
+
+
+
+
+
+        
+
```
Comment (sharmayatendra) [inline documentation.html:727]: you can take care of whitespace & indentation as well

**Review by sharmayatendra — COMMENTED** [review]
good work.. can look upon suggested changes

File: css/card.css (hunk 1 above)
Comment (yogesh-) [reply css/card.css]: resolved, replaced with hex

File: documentation.html (hunk 3 above)
Comment (yogesh-) [reply documentation.html:456]: ok

File: documentation.html (hunk 4 above)
Comment (yogesh-) [reply documentation.html:556]: okay

File: css/card.css (hunk 2 above)
Comment (yogesh-) [reply css/card.css]: image

Done.

**Pull request merged**